core/sequencer/sequencer.go

```diff
@@ -132,13 +132,14 @@ func (s *Sequencer) StartSequencingAll(ctx context.Context, refresh time.Duratio
 			return fmt.Errorf("admin.List(): %v", err)
 		}
 		for _, d := range domains {
-			if _, ok := started[d.Domain]; !ok {
+			if !started[d.Domain] {
 				glog.Infof("StartSigning domain: %v", d.Domain)
 				started[d.Domain] = true
 				go s.StartSigning(ctx, d.LogID, d.MapID, d.MinInterval, d.MaxInterval)
 			}
 		}
 	}
+	ticker.Stop()
```

Comment:
It won't be stopped in the early-out return "admin.List(): ... " above.
How about this:

defer func() {
  ticker.Stop()
}()

immediately after making the new ticker?

Reply:
good point